### PR TITLE
Ubuntu: now supporting 20.04

### PR DESCRIPTION
**Pull request by bryanlatten**

### CODE REVIEW

File: Dockerfile-ubuntu-20.04 [hunk 1]
```diff
@@ -0,0 +1,53 @@
+FROM ubuntu:20.04 as base
```
Comment (mannytoledo) [inline Dockerfile-ubuntu-20.04:1]: For other reviewers, this line is the only difference between this file and the 18.04 version.

Comment (bryanlatten) [reply Dockerfile-ubuntu-20.04:1]: ...thankfully!

File: container/root/scripts/install_s6.sh [hunk 2]
```diff
@@ -14,13 +14,9 @@ curl -fL https://github.com/just-containers/s6-overlay/releases/download/${S6_VE
...
-# Special handling - CentOS
+# Special handling - CentOS >= 7 + Ubuntu >= 20.04
 # @see https://github.com/just-containers/s6-overlay#bin-and-sbin-are-symlinks
-if [ -f /etc/centos-release ]; then
-  tar xzf /tmp/${S6_NAME} -C / --exclude="./bin"
-  tar xzf /tmp/${S6_NAME} -C /usr ./bin
-else
-  tar xzf /tmp/${S6_NAME} -C /
-fi
+tar xzf /tmp/${S6_NAME} -C / --exclude="./bin"
```
Comment (bryanlatten) [inline container/root/scripts/install_s6.sh:19]: symlinks for `bin/bash` did break shell without this fix. It is backwards compatible

Comment (sravankasu) [reply container/root/scripts/install_s6.sh:19]: @bryanlatten Does this apply to alpine images as well?

Comment (bryanlatten) [reply container/root/scripts/install_s6.sh:19]: yes, it applies to all

**Comment by sravankasu** [comment]
LGTM 👍

File: container/root/scripts/install_s6.sh (hunk 2 above)
Comment (ko-be) [reply container/root/scripts/install_s6.sh:19]: If we can guarantee that we're using GNU tar - we can combine these 2 tar commands with:

```
tar xzf /tmp/${S6_NAME} -C / --keep-directory-symlink
```

Comment (ko-be) [reply container/root/scripts/install_s6.sh:19]: I just checked BSD tar and it does not have `--keep-directory-symlink` so I think leaving it w/ 2 tar commands would be best.

**Pull request merged**